src/vcProject.cpp

```diff
@@ -124,20 +124,39 @@ udError   vcProject_CreateFileScene(vcState *pProgramState, const char *pFileNam
   if (pProjectName == nullptr || pProjectName[0] == '\0')
     return udE_InvalidParameter;
 
-  if (udFileExists(pFileName) == udR_Success)
-    return udE_WriteFailure;
-
-  udProject *pNewProject = nullptr;
-  udError error = udProject_CreateInFile(&pNewProject, pProjectName, pFileName);
-  if (error != udE_Success)
-    return error;
+  udError error = udE_Success;
+  udProject* pNewProject = nullptr;
+  bool overrideProject = false;
 
+  if (udFileExists(pFileName) == udR_Success)
+  {
+    overrideProject = true;
+    error = udProject_CreateInMemory(&pNewProject, pProjectName);
+    if (error != udE_Success)
+      return error;
+  }
+  else
+  {
+    error = udProject_CreateInFile(&pNewProject, pProjectName, pFileName);
+    if (error != udE_Success)
+      return error;
+  }
   if (pProgramState->activeProject.pProject != nullptr)
     vcProject_Deinit(pProgramState, &pProgramState->activeProject);
 
   pProgramState->activeProject.pProject = pNewProject;
   vcProject_InitScene(pProgramState, srid);
 
+  if (overrideProject)
+  {
+#if UDPLATFORM_WINDOWS
```

Comment:
This check isn't necessary, `pProgramState->settings.window.useNativeUI` will be `false` on all platforms that aren't supported. By putting in this explicit check, you're creating future bugs as we expand the native UI support.

Reply:
Fixed